lib/tasks/upgrade.rake

```diff
@@ -48,6 +49,11 @@ namespace :cg do
       User.includes(:keys, :public_profile, :private_profile).find_each(&:migrate_permissions!)
     end
 
+    desc "Upgrade passwords to bcrypt(sha1) digests"
+    task(:secure_password => :environment) do
+      User.where(password_digest: nil, type: nil).find_each(&:wrap_old_password)
+    end
```

Comment:
I'm not sure what you mean by dropping the field. It will be set to nil by wrap_old_password. In terms of removing the field from the database I still kept a few legacy fields around in other places as well. I want to clean them up once the migration to the new server is done and we confirmed things still work.